.github/workflows/publish.yml

```
@@ -82,3 +82,15 @@ jobs:
         run: npx vsce publish --no-git-tag-version --packagePath $(find . -iname *.vsix)
         env:
           VSCE_PAT: ${{ secrets.VSCE_PAT }}
+
+  publish-ovsx:
+    name: Publish All (OpenVSX)
+    runs-on: ubuntu-latest
+    needs: build
+    if: success() && startsWith( github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/download-artifact@v3
+      - name: Publish Extension
+        run: npx ovsx publish --no-git-tag-version --packagePath $(find . -iname *.vsix)
```

```suggestion
        run: npx ovsx publish --packagePath $(find . -iname *.vsix)
```

This flag is actually not supported in ovsx from what I can tell.

---

I am not sure myself, but I suppose if removing it doesn't break anything, should be fine.